x-pack/plugins/security_solution/public/resolver/view/assets.tsx

```diff
@@ -309,39 +298,52 @@ const SymbolsAndShapes = memo(() => (
       <path
         d="M87.52113,24.73352a3.7956,3.7956,0,0,0-1.43182-1.47166L45.91012.17918a3.8365,3.8365,0,0,0-3.82049,0L1.91029,23.26186A3.86312,3.86312,0,0,0-.00009,26.55445V72.7199a3.79834,3.79834,0,0,0,1.91041,3.29249L42.08963,99.09514a3.83689,3.83689,0,0,0,3.82049,0L86.08931,76.01239a3.79852,3.79852,0,0,0,1.91056-3.29249V26.55445A3.77643,3.77643,0,0,0,87.52113,24.73352Z"
         transform="translate(0.00013 0.39551)"
-        fill="#010101"
+        fill={isDarkMode ? '#010101' : '#fff'}
       />
       <g opacity="0.7">
         <path
+          opacity={isDarkMode ? 1 : 0.6}
           d="M87.52113,24.73352a3.7956,3.7956,0,0,0-1.43182-1.47166L45.91012.17918a3.8365,3.8365,0,0,0-3.82049,0L1.91029,23.26186A3.86312,3.86312,0,0,0-.00009,26.55445V72.7199a3.79834,3.79834,0,0,0,1.91041,3.29249L42.08963,99.09514a3.83689,3.83689,0,0,0,3.82049,0L86.08931,76.01239a3.79852,3.79852,0,0,0,1.91056-3.29249V26.55445A3.77643,3.77643,0,0,0,87.52113,24.73352Z"
           transform="translate(0.00013 0.39551)"
           fill={`url(#${PaintServerIds.terminatedProcessCube})`}
         />
         <path
+          opacity={isDarkMode ? 0.3 : 0.4}
           d="M.57134,24.52282a3.79906,3.79906,0,0,1,1.34824-1.32625L42.09878.11387a3.83708,3.83708,0,0,1,3.8208,0L86.09877,23.19655a3.79771,3.79771,0,0,1,1.43182,1.47165L44.00909,49.57182Z"
           transform="translate(0.00013 0.39551)"
           fill="#fff"
-          opacity="0.3"
           style={{ isolation: 'isolate' }}
         />
         <path
+          opacity={isDarkMode ? 0.2 : 0.4}
           d="M43.99972,49.63713V99.60449a3.83406,3.83406,0,0,1-1.91025-.50932L1.91029,76.01239A3.79835,3.79835,0,0,1-.00013,72.7199V26.55445A3.77431,3.77431,0,0,1,.562,24.5882Z"
           transform="translate(0.00013 0.39551)"
           fill="#353944"
-          opacity="0.2"
           style={{ isolation: 'isolate' }}
         />
       </g>
     </symbol>
     <symbol id={SymbolIds.terminatedTriggerCube} viewBox="0 0 88 100">
       <title>{'Terminated Trigger Process'}</title>
-      <path
-        d="M87.52143,25.06372a3.795,3.795,0,0,0-1.43129-1.47166L45.92578.50939a3.83384,3.83384,0,0,0-3.81907,0L1.94219,23.59206A3.8634,3.8634,0,0,0,.03252,26.88465V73.05008a3.7986,3.7986,0,0,0,1.90971,3.2925L42.10671,99.42532a3.83423,3.83423,0,0,0,3.81907,0L86.09014,76.34258A3.79881,3.79881,0,0,0,88,73.05008V26.88465A3.77748,3.77748,0,0,0,87.52143,25.06372Z"
-        transform="translate(0)"
-        fill="#010101"
-      />
+      {isDarkMode && (
```

Comment:
:grey_question: Why does the `path` go inside the group when it's light mode and outside when it's dark? If it's just to get the opacity can you do like a `opacity={darkMode ? 1 : 0.6}`

Reply:
If I remember correctly it was a layering issue and how it played off of the  wrapped opacity within the group tag. I tried to maintain as much similarity to the svg export that @lindseypoli provided. I could probably shift it and recalculate the opacity for dark mode and just have one path item in the group tag?

Reply:
It's an asset :shrug: . Maybe next time you're around here, but it's fine for now.